src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -1148,10 +1151,23 @@ vsiAddFilesToScan :: (Has (Lift IO) sig m, Has Diagnostics sig m) => ApiOpts ->
 vsiAddFilesToScan apiOpts scanID files = fossaReq $ do
   (baseUrl, baseOpts) <- useApiOpts apiOpts
 
-  let body = VSIAddFilesToScanRequestBody files
+  -- This is in the API layer because it's an implementation detail of the API.
+  -- The API expects all file paths to be normalized to forward slashes.
+  let normalized = Map.fromList . normalizeFiles $ Map.toList files
+  let body = VSIAddFilesToScanRequestBody normalized
```

Comment:
[Not blocking] It would be good to have some tests for `vsiAddFilesToScan`

Reply:
Longer term integration testing is handled more cleanly with the `foundation-libs` powered VSI client, but this is intended to be a quicker fix than integrating that. 

Since integration testing is handled more cleanly on that repo I plan to focus my efforts on integrating that instead of implementing integration testing in this repo.

Reply:
Right, you mentioned that in the PR description. Sounds reasonable to me!